src/js/review/SubmitButtons.jsx

```diff
@@ -97,7 +97,7 @@ export default function SubmitButtons(props) {
     if (process.env.NODE_ENV !== 'production') {
       submitButton = (
         <div className="small-6 usa-width-one-half medium-6 columns">
-          <a onClick={onSubmit}>Submit again</a>
+          <button onClick={onSubmit}>Submit again</button>
```

Comment:
I know the Vets.gov team made this a link for a specific reason, and not that we should be beholden to that, but I think this kind of change might need more review. Would be curious for their input on this.

Reply:
It doesn't apply in production so web site users should never see it? Or is the vets.gov site run in development mode? From an a11y point it really is a button and not a link (do something vs go somewhere) so there's that.

Looking at the rest of the file which still has references to the vets.gov site, I figured this was a pretty low-risk change. At the moment it's not possible to submit a form using USFS unless you have a server that works just like the vets.gov one, so this code won't be reached except by vets.gov.

Reply:
Yes, I think it's only a feature used for development and not something real users should ever see. I remember some discussion around this feature awhile back, though I can't remember the reasoning for making it a link, so I just wanted to double check with them in case I'm missing anything.

Reply:
:thinking: I don't remember why we went with an anchor instead of a button. Seems pretty safe to me, though.

Reply:
I think we made this a link for style reasons. It should really be a button. Like Dave says, this is never shown in production, so it's a low-risk change.

Also, this extra button isn't super useful to us anymore, because we used to use it in our staging environment, but that's run in production mode. The conditional used to be for our `buildtype` flag instead of NODE_ENV, which doesn't make sense in a separate library.

Reply:
Can we remove it?

Reply:
If so let's add that as a task to #211. I only made minimal changes in this PR to get eslint to pass. I suppose I could have done that with a disable-next-line comment as well but I'd have to look at whether you can do that inside a JSX block.

Reply:
Most of our forms are wrapped in our save-in-progress functionality, so submitting the form again should be pretty easy--just load the saved form and try again.

I personally haven't used that retry ~~button~~ link in a long time. It's probably safe to remove.